### DISCUSSION TITLE
How can I stream text into the editor?

### DISCUSSION THREAD

**Discussion by trevorpfiz** [ueberdosis/tiptap#4946] [General Questions]
I am using [useCompletion](https://sdk.vercel.ai/docs/api-reference/use-completion) from Vercel AI. How can I stream the completion into the editor?

**Comment by shreeram312** [comment]
@trevorpfiz Have u figured it out..?

**Comment by iamkingsleyf** [comment]
did you get it to work?

**Comment by bdbch** [comment]
Since Tiptap is currently not supporting Markdown yet - your best guess would be to transform the response into HTML and inserting it into the editor via the `insertContent` commands.

**Reply by iamkingsleyf** [reply]
i got it working thanks
